lib/dotcom_web/controllers/schedule/alerts_controller.ex

```diff
@@ -12,8 +12,16 @@ defmodule DotcomWeb.ScheduleController.AlertsController do
   plug(:tab_name)
 
   def show(conn, _) do
+    mode =
+      conn
+      |> Map.get(:assigns, %{})
+      |> Map.get(:breadcrumbs, %{})
+      |> Enum.at(1, %{})
+      |> Map.get(:text)
+
```

Comment:
Interesting, I would've first tried to get this via `Routes.Route.type_atom(conn.assigns.route)`, or maybe `path_atom` or `type_name` from that same module. To me that seems more direct, and that wouldn't be affected if we ever remove breadcrumbs or change what they say.

Reply:
Oh, if there is a better place to get the info, I'll take it!